test/integration/routes.test.js

```diff
@@ -504,6 +525,41 @@ describe('routes', function () {
         expect(runProcessResult.status).to.equal(400)
       })
 
+      test('add item with out of range index for parent', async function () {
+        // add parent to be consumed
+        const firstToken = await postRunProcess(app, authToken, [], [{ roles: defaultRole, metadata: {} }])
+        expect(firstToken.status).to.equal(200)
+        const lastToken = await getLastTokenIdRoute(app, authToken)
+        const firstTokenId = lastToken.body.id
+
+        // add new token with out of range parent_index
+        const inputs = [firstTokenId]
+        const outputs = [{ roles: defaultRole, metadata: {}, parent_index: 99 }]
+        const secondToken = await postRunProcess(app, authToken, inputs, outputs)
+
+        expect(secondToken.body.message).to.contain('out of range')
```

Comment:
I'm assuming this message is dynamic (not predictable, different each time), otherwise you should compare on the on the whole string message result using the expected value.

Reply:
they're not dynamic, I think I thought they were cause so many others are. Setting them to the compare the whole string